tests/testthat/test-package.R

```diff
@@ -82,3 +82,16 @@ test_that("titles don't get autolinked code", {
   rd <- rd_text("\\title{\\code{foo()}}", fragment = FALSE)
   expect_equal(extract_title(rd), "<code>foo()</code>")
 })
+
+test_that("read_meta() errors gracefully if _pkgdown.yml failed to parse", {
+  pkg <- local_pkgdown_site()
```

Comment:
This isn't quite right yet, but we're moving towards more inline tests, where possible.

Reply:
Oh, I see! Let me know if you want me to done further adjustments